nav2_bt_navigator/README.md

```diff
@@ -154,11 +140,11 @@ Image below depicts the graphical version of the complete Navigation Task with A
 
 <img src="./doc/AutoLocalization_w_recovery_parallel.png" title="Navigation Behavior Tree with AutoLocalization, Recovery, and Planning & Control" width="70%" align="middle">
 
+## Future Work
+Scope base failure handling: Behavior Tree architecture allows the recovery node to handle failures at multiple scopes. With multi-layer failure handling, any action in a large system can be constructed with specific recovery actions suitable for that action. Thus, failures in these actions can be handled locally within the scope. With such design, a system can be recovered at multiple levels based on the nature of the failure. Higher level recovery actions could be recovery actions such as re-initializing the system, re-calibrating the robot, bringing the system to a good known state, etc.  Currently, in the navigation stack multi-scope, recovery actions are not implemented. The figure below highlights a simple multi-scope recovery handling for the navigation task.
```

Comment:
"Scope base failure handling" -> "Scope-based failure handling"

"With multi-layer failure handling,..." -> "That is, any action..."  or "Using this capability, any action..."

"Behavior Tree architecture" -> "The Behavior Tree architecture"

"Currently, in the navigation stack multi-scope, recovery actions are not implemented" -> "Currently, in the navigation stack, multi-scope recovery actions are not implemented"